tests/indent/hcl_spec.lua

```diff
@@ -0,0 +1,20 @@
+local Runner = require("tests.indent.common").Runner
+--local XFAIL = require("tests.indent.common").XFAIL
+
+local run = Runner:new(it, "tests/indent/hcl", {
+  tabstop = 2,
+  shiftwidth = 2,
+  expandtab = true,
+})
+
+describe("indent Lua:", function()
```

Comment:
should be HCL

Reply:
:facepalm:

Reply:
The test failure can be found when searching for "indent Lua:" so the test works as expected!